llvm/test/MC/ELF/debug-hash-file-disabled-dwarf.s

```diff
@@ -0,0 +1,12 @@
+// RUN: llvm-mc -triple x86_64-unknown-linux-gnu -filetype obj -o %t %s
+// RUN: llvm-readelf --sections %t | FileCheck %s
```

Comment:
Nit: You can unify this test to `debug-hash-file-empty-dwarf.s` with another prefix.